src/NUnitFramework/framework/Constraints/Numerics.cs

```diff
@@ -418,5 +418,75 @@ public static int Compare(object expected, object actual)
             return Convert.ToInt32(expected).CompareTo(Convert.ToInt32(actual));
         }
         #endregion
+
+        #region Numeric Difference
+
+        /// <summary>
+        /// Calculates the difference between 2 values.
+        /// </summary>
+        /// <param name="expected">The expected value</param>
+        /// <param name="actual">The actual value</param>
+        /// <param name="tolerance">A reference to the tolerance in effect</param>
+        /// <returns>The difference between the values</returns>
+        public static object Difference(object expected, object actual, Tolerance tolerance)
```

Comment:
I think it makes more sense to use `ToleranceMode` and not `Tolerance` (see comment in `NumericsTest.cs`). Could this method be internal?

Reply:
done